tests/Google/Ads/GoogleAds/Lib/V3/GoogleAdsClientBuilderTest.php

```diff
@@ -62,7 +63,8 @@ public function testBuildClientFromConfiguration()
             ['developerToken', 'GOOGLE_ADS', self::$DEVELOPER_TOKEN],
             ['loginCustomerId', 'GOOGLE_ADS', self::$LOGIN_CUSTOMER_ID],
             ['endpoint', 'GOOGLE_ADS', 'https://abc.xyz:443'],
-            ['proxy', 'CONNECTION', 'https://localhost:8080']
+            ['proxy', 'CONNECTION', 'https://localhost:8080'],
+            ['transport', 'CONNECTION', self::provideValidTransports()[0][0]]
```

Comment:
This is better just 'grpc'.
We don't benefit from using `provideValidTransports` here, so using it make it harder to read this code, especially with double indices [0].